firmware/library/L3_HAL/Piezo_Driver.hpp

```diff
@@ -0,0 +1,39 @@
+// This driver is a layout for the piezo buzzer controller implementation
+// It contains the interface, which is inherited by the "Buzzer" class
+// which implements the functions in the interface
+// The constructor is responsible for initializing the port
+
+#pragma once
+
+// import only the necessary libraries to make this interface function
+#include <cstdint>
+
+// import the lpc_pwm class from SJOne library as our PWM driver
+#include "/Users/mianhashimshah/OpenSJ_Bluz/SJOne/L2_Drivers/lpc_pwm.hpp"
```

Comment:
This will break if you include the file like this. Don't use absolute directory paths that include specifics about your directory.